server/db/migrations/20_create_grid_service_instance.rb

```diff
@@ -1,7 +1,12 @@
 class CreateGridServiceInstance < Mongodb::Migration
   def self.up
     GridServiceInstance.create_indexes
-    Container.unscoped.where(container_type: 'container').includes(:grid_service).each do |c|
+    migrate_containers('volume')
+    migrate_containers('container')
```

Comment:
In theory yes, but I'm not able to figure out a scenario where we would have ended up in situation where there is multiple volume containers for a given instance.